mender.go

```diff
@@ -64,8 +64,9 @@ const (
 )
 
 var (
-	defaultManifestFile = path.Join(getConfDirPath(), "build_mender")
-	defaultDataStore    = getStateDirPath()
+	defaultArtifactInfoFile = path.Join(getConfDirPath(), "artifact-info")
+	defaultDeviceTypeFile   = path.Join(getStateDirPath(), "device_type")
```

Comment:
Yes, it should be `artifact_info`. Not sure why I didn't do it immediately, it's the only one with a dash right now. I will change it.